collab2/foraging/toolkit/utils.py

```diff
@@ -0,0 +1,186 @@
+import math
+from itertools import product
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from scipy.signal import find_peaks
+import warnings
+
+#define a class to streamline object creation 
+class dataObject:
+    def __init__(self, foragersDF, grid_size=None, rewardsDF=None, frames=None):
```

Comment:
I know I often didn't in the past, but let's try to do better. :) - please add type annotations

Reply:
Done

Reply:
where?::>